### PR TITLE
Add update state v2 for issuer and holder

### PR DESCRIPTION
**Pull request by mirgee**
Signed-off-by: Miroslav Kovar <miroslavkovar@protonmail.com>

### CODE REVIEW

File: agents/node/vcxagent-core/test/issuer-update-state-v2.spec.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    await aliceVcxControl.waitForCredentialOfferAndAccept(connectionIdAtAlice, true)
+
+    logger.info('Shutting down Alice VCX session.')
+    await shutdownVcx()
+  }
+
+  return { acceptInvite, acceptCredential }
+}
+
+describe('test update state', () => {
```
Comment (Patrik-Stas) [inline agents/node/vcxagent-core/test/issuer-update-state-v2.spec.js:142]: beautiful suite 👍

**Comment by Patrik-Stas** [comment]
Looks great! 👍  but still missing v2_update for `proof` and `disclosedProof`

**Comment by mirgee** [comment]
@Patrik-Stas I thought of separating them into different PRs (as suggested by the name of this PR), but may do all in one if that's helpful in any way.

**Comment by Patrik-Stas** [comment]
@mirgee okay then, let's do it in separate PR. However one more thing here is that you forgot to update `vcxclient-interactive.js` according to new `createVcxAgent` API

**Comment by mirgee** [comment]
@Patrik-Stas What changes do you mean? The API is unchanged?

**Comment by Patrik-Stas** [comment]
@mirgee I mean this change:
from 
```
async function createVcxAgent (storageService, logger) {
```
to
```
async function createVcxAgent ({ agentName, protocolType, agencyUrl, seed, webhookUrl, usePostgresWallet, logger, rustLogLevel })
```
And file `vcxclient-interactive.js` is still assuming the former signature

**Pull request merged**